apps/widget-configurator/vite.config.ts

```diff
@@ -14,24 +17,31 @@ const plugins = [
     },
     protocolImports: true,
   }),
+  macrosPlugin(),
```

Comment:
why did you need macros in this PR?

Reply:
<img width="506" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/5b07376b-fd75-4e77-99ed-8865a4b66e0f">

Without the plugin we get the error

Reply:
I see it came from `ms.macro`